src/test/libslic3r/test_3mf.cpp

```diff
@@ -0,0 +1,20 @@
+#include <catch.hpp>
+#include <test_options.hpp>
+#include <string>
+#include "Model.hpp"
+#include "TMF.hpp"
+
+
+using namespace Slic3r;
+
+SCENARIO("Reading 3mf file") {
+    GIVEN("umlauts in the path of the file") {
+         auto model {new Slic3r::Model()};
+        WHEN("3mf model is read") {
+            auto ret = Slic3r::IO::TMF::read(std::string(testfile_dir) +"test_3mf/Geräte/box.3mf",model);
```

Comment:
Use 
````
testfile("test_3mf/Geräte/box.3mf")
````
Instead.

Reply:
https://github.com/slic3r/Slic3r/blob/master/src/test/test_options.hpp.in

Reply:
https://github.com/slic3r/Slic3r/blob/master/src/test/test_options.hpp.in

Reply:
I followed test_config.cpp this line `std::string(testfile_dir) +` was used there
but I fixed it now :)